### PR TITLE
Init issue templates

### PR DESCRIPTION
**Pull request by mvaivre**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: 🤔 Questions and Help
+    url: https://discord.com/channels/747741246667227157/883355707146141737
```
Comment (polarker) [inline .github/ISSUE_TEMPLATE/config.yml:3]: I created this permanent invite link for support/general channel: https://discord.gg/dT3hDTtABD

Comment (mvaivre) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Perfect! Thanks.

**Review by polarker — APPROVED** [review]
Great one ❤️

**Comment by mvaivre** [comment]
You can see the template in action here: https://github.com/mvaivre/alephium-wallet/issues/new/choose

**Pull request merged**